### PR TITLE
Standardized the usage docs for cf-icons

### PR DESCRIPTION
**Pull request by jimmynotjim**
Updated usage documentation for cf-icons to match the formatting and structure of #341 and #347.

## Additions

- Table of Contents

## Changes

- Standardized headings and org of description text

## Testing

- Test on your local markup preview tool

## Review

- @KimberlyMunoz 
- @contolini 
- @cfpb/front-end-team-admin 

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/cf-icons/usage.md [hunk 1]
```diff
@@ -45,53 +68,55 @@ on i elements and then another font for the icons.
 Note that this issue only pops up in older versions of Internet Explorer.
 
 
-## Helper classes for icon sizes
+## Helpers
+
+### icon sizes
```
Comment (KimberlyMunoz) [inline src/cf-icons/usage.md]: Title case.

File: src/cf-icons/usage.md [hunk 2]
```diff
@@ -125,7 +150,7 @@ MIT Licensed by Font Awesome
 
 Border color set by @cf-icon-border-color
```
Comment (KimberlyMunoz) [inline src/cf-icons/usage.md]: Tickmarks around classname

File: src/cf-icons/usage.md [hunk 3]
```diff
@@ -100,7 +125,7 @@ MIT Licensed by Font Awesome
 First parameter is @degrees.
```
Comment (KimberlyMunoz) [inline src/cf-icons/usage.md]: Tickmarks around classname.

**Comment by jimmynotjim** [comment]
@KimberlyMunoz Ready for review again

**Comment by KimberlyMunoz** [comment]
:+1: :shipit:

**Comment by contolini** [comment]
If you want these changes to make their way into npm, you'll need to add a line item in `CHANGELOG.md`. And don't feel obligated -- It's totally fine to not update the changelog if you feel these are insignificant documentation updates. They'll get published to npm the next time cf-icons is updated.

**Comment by jimmynotjim** [comment]
@contolini Updated. I typically wait so I don't have to resolve conflicts since that config flag doesn't seem to work as well as we had hoped. Is there a way to have the bot check for a change in the changelog before automerging? That way if you get two sign offs but leave off the changelog, you can just push that update and it'll be merged then?

**Comment by contolini** [comment]
@jimmynotjim add `[MAJOR|MINOR|PATCH]` after the component's name in the changelog. (Virtually everyone forgets to do this and it's indicative that we need to simplify our CHANGELOG process).

**Comment by jimmynotjim** [comment]
@contolini dang it, I knew I was forgetting something.

**Comment by contolini** [comment]
👍

**Pull request merged**